README.md

```diff
@@ -44,23 +44,36 @@ We have already implemented some pieces of this project:
 
 ## Wave 1
 
-Update `Square`, `Board` and `App.js` to respond to click events on the squares.  
+Update the `Board` component to render the grid of squares.  You will need to complete
 
-Each `Square` component should take 3 props.
+- `generateSquareComponents` in the `Board` component
```

Comment:
Do we still want this to be a bullet even though it's only one thing? I think it might read better just as one sentence but maybe that's just me.